docs/linux/sql-server-linux-management-overview.md

```diff
@@ -28,6 +28,10 @@ Almost everything you can do in a client tool can also be accomplished with Tran
 
 The new Azure Data Studio is a cross-platform tool for managing SQL Server. For more information, see [Azure Data Studio](../azure-data-studio/what-is-azure-data-studio.md).
 
+## Named Pipes
```

Comment:
Can we phrase this as " Named pipes protocol is not supported for SQL Server on Linux."